plik1.txt

```diff
@@ -0,0 +1 @@
+hello my mumi
```

Comment:
You must insert content "Hello my mum".
Please improve this issue, and get me now immediately, if you do this.